annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -141,6 +148,56 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
        return getVariantAnnotation(variants, isoformOverrideSource);
     }
 
+    @ApiOperation(value = "getHotspotAnnotation",
```

Comment:
Would it be a good idea to include hotspot annotation by default when getting variants? I'm thinking about how this is going to be used in the pipelines - we already make two calls to genome-nexus for the annotation and transcripts. Now we would have to make a third for the hotspot annotation. I'm sort of new to looking at this code, but I feel like combining the functionality of some of these endpoints might be a good idea.